libs/vm_modules/tests/unit/state_tests.cpp

```diff
@@ -195,6 +195,19 @@ TEST_F(StateTests, querying_state_constructed_from_null_string_fails_gracefully)
   ASSERT_FALSE(toolkit.Run(&output));
 }
 
+TEST_F(StateTests, serialising_compound_object_with_null_values_does_not_segfault)
```

Comment:
👍
I actually test for this for segfault fix PR, but then after messing with merges back and forth it did't make it in to the PR for some reason.